Pod/Classes/SEGMixpanelIntegration.m

```diff
@@ -139,6 +140,32 @@ - (void)screen:(SEGScreenPayload *)payload
     }
 }
 
+
+- (void) group:(SEGGroupPayload *)payload {
+    
+    NSString *groupID = payload.groupId;
+    NSDictionary *traits = [payload traits];
+    NSString *groupName = traits[@"name"];
+    
+    NSArray *groupIdentifierProperties = [self.settings objectForKey:@"groupIdentifierTraits"];
+    
+    if(groupName == nil || groupName.length == 0 || groupIdentifierProperties.count == 0) {
```

Comment:
We can change this to just be `!groupID || groupIdentifierProperties.count == 0`. And then delete the reference to `groupName` on line 148 since we are not using groupName anymore as the identifier and we are relying on the `groupIdentifierProperties`.

Reply:
Ohh yes, sorry my bad. Changing right away.